packages/components/src/components/Snackbar/Snackbar.tsx

```diff
@@ -48,17 +72,222 @@ type SnackbarProviderProps = {
  *
  * The Snackbar remains open indefinitely. App configuration should ensure it is dismissed on navigation.
  */
-export const SnackbarProvider: React.FC<SnackbarProviderProps> = (
-  {
-    // globalSnackbar,
-  },
-) => {
+export const SnackbarProvider: React.FC = () => {
+  // const forceUpdate = useReducer(x => x + 1, 0)[1]
+  // const [offset, setOffset] = useState(10)
+  // useEffect(
+  //   () => {
+  //     if (offset != globalThis.snackbarOffset) {
+  //       globalThis.updateSnackbarOffset(globalThis.snackbarOffset)
+  //       forceUpdate()
+  //       return
+  //     }
+  //   }, [offset]
+  // )
+
+  // useEffect(() =>
+  //   AccessibilityInfo.announceForAccessibility('Test announce provider'),
+  // )
+
   return (
     <Toast
       animationDuration={100}
       duration={1000000000000} // Essentially indefinite until dismissed
+      offset={50}
+      placement="bottom"
       ref={(ref) => ((globalThis.snackbar as ToastContainer | null) = ref)}
+      renderToast={(toast) => <Snackbar {...toast} />}
       swipeEnabled={false}
     />
   )
 }
+
+type SnackbarButtonProps = {
+  text: string
+  onPress: () => void
+}
+
+const SnackbarButton: FC<SnackbarButtonProps> = ({ text, onPress }) => {
+  const theme = useTheme()
+
+  const helperTextBold: TextStyle = {
+    fontFamily: 'SourceSansPro-Bold',
+    fontSize: 16,
+    lineHeight: 22,
+  }
+  const typography = { helperTextBold }
```

Comment:
Is this const needed? Seems like it's only used once and could just do `...helperTextBold` below.

Reply:
No, just making it structured the same way it'll be when we have the typography tokens--but guess maybe that's taking an unnecessary step since helperTextBold probably won't exist as a typography token anyway.

I can remove it.